core/common/src/main/java/alluxio/util/FormatUtils.java

```diff
@@ -198,6 +198,37 @@ public static long parseSpaceSize(String spaceSize) {
   }
 
   /**
+   * Parses a String size to Milliseconds.
+   *
+   * @param timeSize the size of a time, e.g. 1M, 5H, 10D
+   * @return the time size in milliseconds
+   */
+  public static long parseTimeSize(String timeSize) {
+    double alpha = 0.0001;
+    String time = timeSize.replaceAll("[a-zA-Z]", "");
```

Comment:
the current approach is a bit hacky. The more conventional way to do this is something like this:

```java
Pattern p = Pattern.compile("(\d+)([a-zA-Z]+); // you can define this as a static member in this util class"
Matcher m = p.matcher(timeSize);
if (m.matches()) {
  time = m.group(1);
  time = m.group(2);
}
```